internal/controller/storageclient_controller.go

```diff
@@ -406,68 +405,50 @@ func (r *StorageClientReconciler) newExternalClusterClient() (*providerClient.OC
 }
 
 // onboardConsumer makes an API call to the external storage provider cluster for onboarding
-func (r *StorageClientReconciler) onboardConsumer(externalClusterClient *providerClient.OCSProviderClient) (reconcile.Result, error) {
+func (r *StorageClientReconciler) onboardConsumer(externalClusterClient *providerClient.OCSProviderClient) error {
 
 	// TODO Need to find a way to get rid of ClusterVersion here as it is OCP
 	// specific one.
 	clusterVersion := &configv1.ClusterVersion{}
 	clusterVersion.Name = "version"
 	if err := r.get(clusterVersion); err != nil {
 		r.Log.Error(err, "failed to get the clusterVersion version of the OCP cluster")
-		return reconcile.Result{}, fmt.Errorf("failed to get the clusterVersion version of the OCP cluster: %v", err)
+		return fmt.Errorf("failed to get the clusterVersion version of the OCP cluster: %v", err)
 	}
```

Comment:
We don't need to fetch the clusterVersion anymore, it was used to decide the name of the storageConsumer

Reply:
done.